validator/sawtooth_validator/database/lmdb_nolock_database.py

```diff
@@ -54,6 +54,7 @@ def __init__(self, filename, flag):
                                       create=create,
                                       lock=True)
 
+    # pylint: disable=no-value-for-parameter
```

Comment:
Not in any obvious way. The thing works as far as I can tell, so the error might be spurious. In any case, I've alerted @jsmitchell, who can take further action if need be.